app/models/attendee.js

```diff
@@ -20,7 +20,6 @@ export default ModelBase.extend({
   /**
    * Relationships
    */
-  order  : belongsTo('order'),
```

Comment:
If I include this then attendee do not serialize on orders request. Removing this will affect attendees table. 
i tried a lot but could not figure out the problem. I will search for appropriate solution for why this happens.